var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -446,7 +446,7 @@ def determine_b2_options(self, spec, options):
             '--layout=%s' % layout
         ])
 
-        if not spec.satisfies('%intel'):
+        if (not spec.satisfies('%intel')) or spec.satisfies('@1.76:'):
```

Comment:
```suggestion
        if not spec.satisfies('@:1.75 %intel')):
```
Is maybe a bit more concise? Did you check only 1.76 or also older versions? 

For reference this line was added in 2016: https://github.com/spack/spack/pull/1816#issuecomment-248991860, there's been enough releases since...

Reply:
> Did you check only 1.76 or also older versions?

It might go back farther, but I've only checked for `1.76`